### ISSUE TITLE
Add Decision-related columns to the database view

### ISSUE THREAD

**Issue by AARON-CLARK** [pharmaR/riskassessment#441]
Add two new columns to the database view:
* Decision Date"
* "Decision by" which should contain a username, and when automation dictated the decision, including "auto-decision" as the user

**Comment by Robert-Krajcik** [comment]
I am looking at this and doing some planning.

`Decision Date` and `Decision by` are at the package level, so they should be added to the `package` table.

```
CREATE TABLE IF NOT EXISTS package (
   id             INTEGER PRIMARY KEY AUTOINCREMENT,
   name           CHAR, 
   version        CHAR,
   title          CHAR,
   description    TEXT,
   maintainer     CHAR,
   author         CHAR,
   license        CHAR,
   published_on   CHAR,
   score          REAL,
   weighted_score REAL,
   decision       CHAR,
```
   **_`decision_date DATE`_**,  <-- `user$name`
   **_`decision_by  CHAR`_**,    <-- `Sys.Date()`
```
   date_added     DATE
); 
```

These will get updated in the `mod_sidebar.R` module, in the same `observeEvent()` blocks that `Decision` is updated.

```{r, eval=FALSE}
    # Update database info after decision is submitted.
    observeEvent(input$submit_confirmed_decision, {
      dbUpdate(glue::glue(
        "UPDATE package
          SET decision = '{input$decision}'
          WHERE name = '{selected_pkg$name}'")
      )
```
```{r, eval=FALSE}
    observeEvent(input$reset_confirmed_decision, {
      dbUpdate(glue::glue(
        "UPDATE package
          SET decision = ''
          WHERE name = '{selected_pkg$name}'")
      )
```
Lastly, the `was_decision_made` column is redundant.
If the `decision` and `decision_date` columns are blank, then no decision has been made.
I plan to remove it so as to save some space.

**Comment by Robert-Krajcik** [comment]
And we need to update some of the tests, especially the ones that use a copy of `database.sqlite`
and test for things like
`  expect_equal(names(pkg), c("id", "name", "version", "title", "description", "maintainer", "author", "license", "published_on", "score", "weighted_score", "decision", "decision_by", "decision_date", "date_added"))`

**Comment by AARON-CLARK** [comment]
closed with #467
